lib/dotcom/system_status/groups.ex

```diff
@@ -0,0 +1,337 @@
+defmodule Dotcom.SystemStatus.Groups do
+  @moduledoc """
+  A module that groups alerts into statuses for the system status widget.
+  """
+
+  alias Alerts.Alert
+
+  # &groups/2 returns an ordered data structure, sorted in the order
+  # given by `@lines`.
```

Comment:
This comment should be in a `@doc` above the `def` line

Reply:
I should restructure this comment to make it clearer that it's actually about `@lines`, not `groups/2`